marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+
+bool   healthy
+string message
```

Comment:
Those look reasonable. What would you expect the output of a path planning node be in the case where the GPS input is invalid? It seems like a point of confusion because the node itself is happy, but can't operate because it's inputs are invalid. I think a similarly weird situation could happen with a GPS receiver that can't get or maintain a good satellite fix because of environmental conditions. The receiver itself is fine, but the NavSatFix message would tell you there's a problem. Would we expect the health message to then say there's a problem?